From: Iustin Pop <iustin@google.com>
Subject: [PATCH stable-2.5] Fix LVM volume listing with newer LVM
 Per commit 0304f0e, newer LVM has extended the lv_attr field. However,
 that commit was incomplete as we examine this attribute in another
 place in the code.

 Thanks to user alperhome, the _LVSLINE_REGEX in lib/backend.py also
 needs fixing. I've used the same change as in the above commit: accept
 at minimum 6 characters, but allow for more.
 ---
 lib/backend.py |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/lib/backend.py b/lib/backend.py
index 859f94c..a76157f 100644
--- a/lib/backend.py
+++ b/lib/backend.py
@@ -1,7 +1,7 @@
 #
 #
 
-# Copyright (C) 2006, 2007, 2008, 2009, 2010, 2011 Google Inc.
+# Copyright (C) 2006, 2007, 2008, 2009, 2010, 2011, 2012 Google Inc.
 #
 # This program is free software; you can redistribute it and/or modify
 # it under the terms of the GNU General Public License as published by
@@ -77,7 +77,7 @@ _IES_PID_FILE = "pid"
 _IES_CA_FILE = "ca"
 
 #: Valid LVS output line regex
-_LVSLINE_REGEX = re.compile("^ *([^|]+)\|([^|]+)\|([0-9.]+)\|([^|]{6})\|?$")
+_LVSLINE_REGEX = re.compile("^ *([^|]+)\|([^|]+)\|([0-9.]+)\|([^|]{6,})\|?$")
 
 
 class RPCFail(Exception):
-- 
1.7.10

